.travis.yml

```diff
@@ -17,6 +17,17 @@ env:
 
 jobs:
   include:
+    - dist: trusty
+      compiler: i586-mingw32msvc-gcc
+      addons:
+        apt:
+          packages:
+            - mingw32
+            - wine
```

Comment:
Now I spotted that in packages is missing `cmake`. Job may start failing in future if (recycled) image would not contain it.

Reply:
Because this mr was already merged, the change needs to be submitted with a different mr.